clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/CacheResultBuilder.groovy

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
Actually, maybe that isn't minor, does this get picked up as the Javadoc for the class?